spikingjelly/activation_based/model/__init__.py

```
@@ -0,0 +1 @@
+from .spikformer import *
```

Ruff (0.15.12)

[error] `from .spikformer import *` used; unable to detect undefined names

(F403)